includes/misc-functions.php

```diff
@@ -795,3 +795,36 @@ function getallheaders() {
 	}
 
 endif;
+
+/**
+ * Determines the receipt visibility status
+ *
+ * @return bool Whether the receipt is visible or not.
+ */
+function edd_user_can_view_receipt( $payment_key ) {
```

Comment:
Can we change this to `edd_can_view_receipt( $payment_key = '' )`?

- Not everyone viewing a receipt is technically a `user`
- Reads better semantically
- Sets a default value